content/intro/3.md

```diff
@@ -12,7 +12,3 @@ This will load a schema into Dgraph for us to use in the first steps of the
 tutorial. Don't worry too much about the syntax for now, we'll cover all that
 in later lessons, just check in the response panel that the operation was a
 success.
-
-Note that, as we just installed it, Dgraph won't be persisting any
```

Comment:
May be we reword it without removing it. Docker doesn't persist data across restarts if volume is not used.

Reply:
The docker command used in the tutorial is:

```
docker run -it -p 8081:8081 -p 8080:8080 -p 9080:9080 -v ~/dgraph:/dgraph --name dgraph dgraph/dgraph dgraph zero --port_offset=-2000
```

Isn't the `-v` flag already present here?

Reply:
Didn't notice :)